src/endpoints/nfts/entities/nft.ts

```diff
@@ -20,7 +20,7 @@ export class Nft {
   @ApiProperty({ type: String })
   collection: string = '';
 
-  @ApiProperty({ type: Number, nullable: true })
+  @ApiProperty({ type: Number, nullable: true , required: false })
```

Comment:
timestamp should be required for nft ( creation date )

Reply:
fixed